### PR TITLE
Attendee Calendar wowl2

### PR DESCRIPTION
**Pull request by Williambraecky**

### CODE REVIEW

File: addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            // Decline event
+            this.orm.call(
+                "calendar.attendee",
+                "do_decline",
+                [record.attendeeId],
+            ).then(this.model.load.bind(this.model));
+        }
+    }
+
+    configureCalendarProviderSync(ProviderName) {
```
Comment (Williambraecky) [inline addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.js:42]:
```suggestion
    configureCalendarProviderSync(providerName) {
```

File: addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                    // Records will share the same rawRecord.
+                    const record = { ...event };
+                    const attendeeInfo = data.attendees.find(a => (
+                        a.id === attendee &&
+                        a.event_id === event.id
+                    ));
+                    record.attendeeId = attendee;
+                    // Colors are linked to the partner_id but in this case we want it linked
+                    // to attendeeId
+                    // TODO: for color support => `convertRecordToEvent`
```
Comment (Williambraecky) [inline addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js:104]:
```suggestion
```

File: addons/calendar/static/src/views/attendee_calendar/attendee_calendar_popover.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/** @odoo-module **/
```
Comment (Williambraecky) [inline addons/calendar/static/src/views/attendee_calendar/attendee_calendar_popover.js:1]: Delete file

**Review by Williambraecky — COMMENTED** [review]
Self review 🤡

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**